spec/lita/handler/chat_router_spec.rb

```diff
@@ -194,7 +194,7 @@ def self.name
 
   context "when the handler raises an exception" do
     it "calls the error handler with the exception as argument" do
-      expect(registry.config.robot.error_handler).to receive(:call).with(instance_of(TypeError))
+      expect(Lita.config.robot.error_handler).to receive(:call).with(instance_of(TypeError))
```

Comment:
Hmm. As soon as I get a chance I will try it out and see if I can figure out what's going on there.